sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/CapacityReservationGroupsOperations.cs

```diff
@@ -0,0 +1,1620 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.Compute
+{
+    using Microsoft.Rest;
+    using Microsoft.Rest.Azure;
+    using Models;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// CapacityReservationGroupsOperations operations.
+    /// </summary>
+    internal partial class CapacityReservationGroupsOperations : IServiceOperations<ComputeManagementClient>, ICapacityReservationGroupsOperations
+    {
+        /// <summary>
+        /// Initializes a new instance of the CapacityReservationGroupsOperations class.
+        /// </summary>
+        /// <param name='client'>
+        /// Reference to the service client.
+        /// </param>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        internal CapacityReservationGroupsOperations(ComputeManagementClient client)
+        {
+            if (client == null)
+            {
+                throw new System.ArgumentNullException("client");
+            }
+            Client = client;
+        }
+
+        /// <summary>
+        /// Gets a reference to the ComputeManagementClient
+        /// </summary>
+        public ComputeManagementClient Client { get; private set; }
+
+        /// <summary>
+        /// The operation to create or update a capacity reservation group. Please
+        /// refer to https://aka.ms/CapacityReservation for more details. When updating
```

Comment:
>https://aka.ms/CapacityReservation  [](http://example.com/codeflow?start=21&length=35)

Doesn't see any existing link. Is this going to be valid link ?

Reply:
yeah currently this doesnt linked to any public documentation. But once varun create the public documentation for the CR, we can link this aka.ms link to that public doc page.